script/misc/update-tag2link

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+#
+# Run with ./script/misc/update-tag2link config/tag2link.yml
+#
+
+require 'net/http'
+require 'json'
+require 'psych'
+
+QUERIES = {
+  "https://sophox.org/sparql" => "
+    SELECT (CONCAT('Key:', ?permanent_key_ID) as ?OSM_key) ?formatter_URL WHERE {
+      ?item osmdt:P2 osmd:Q7.
+      ?item osmdt:P16 ?permanent_key_ID.
+      ?item osmdt:P8 ?formatter_URL.
+    }
+  ",
+  "https://query.wikidata.org/sparql" => "
+    SELECT ?OSM_key ?formatter_URL WHERE {
+      { ?item wdt:P1282 ?OSM_key. }
+      FILTER(STRSTARTS(?OSM_key, 'Key:')) .
+      {
+        { ?item wdt:P1630 ?formatter_URL. }
+        UNION
+        { ?item wdt:P3303 ?formatter_URL. }
+      }
+    }
+  "
+}
+
+results = QUERIES.map do |(url, query)|
+  uri = URI(url)
+  params = { query: query }
+  uri.query = URI.encode_www_form(params)
+
+  req = Net::HTTP::Get.new(uri)
+  req['Accept'] = "application/sparql-results+json"
+  req['User-Agent'] = "OpenStreetMap Website"
+
+  res = Net::HTTP.start(uri.hostname, uri.port, :use_ssl => uri.scheme == 'https') do |http|
+    http.request(req)
+  end
+
+  if res.code == "200"
+    JSON.parse(res.body)["results"]["bindings"]
+  else
+    puts "HTTP code for #{url} is #{res.code}"
+    exit 1
+  end
+end
+
+filecontent = results.flatten({}) do |memo, result|
+  memo[result["OSM_key"]["value"].slice("Key:".size...)] = result["formatter_URL"]["value"]
+  memo
+end.to_yaml
+
+File.write(ARGV[0], filecontent)
```

Comment:
Can you sort the results by key before generating the file? This way it's easier to merge changes later on.

Reply:
Done.